projects/Allwinner/patches/linux/0040-a64-increase-mali-frequency.patch

```diff
@@ -11,9 +11,9 @@ Signed-off-by: Jernej Skrabec <jernej.skrabec@siol.net>
 --- a/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
 +++ b/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
 @@ -1141,6 +1141,9 @@
- 			clocks = <&ccu CLK_BUS_GPU>, <&ccu CLK_GPU>;
  			clock-names = "bus", "core";
  			resets = <&ccu RST_BUS_GPU>;
+ 			operating-points-v2 = <&gpu_opp_table>;
 +
 +			assigned-clocks = <&ccu CLK_GPU>;
 +			assigned-clock-rates = <432000000>;
```

Comment:
This patch should be removed. Operating points replaces assigned clock rate property.

Reply:
👍 done